docs/manuals/run-deploy/deploy/taipy-cloud/index.md

```diff
@@ -1,4 +1,151 @@
-Taipy Cloud is a platforms that allows for deploying Taipy applications on virtual machines hosted
-remotely.
+Welcome to Taipy Cloud, the platform designed to make application deployment easier,
+ensuring accessibility and stability. In this detailed guide, we will take you through
+the process of deploying a Taipy application on Taipy Cloud.
 
-Documentation is yet to come, when Taipy Cloud is opened.
+![Taipy Cloud](images/logo_artwork.png){width=90% : .tp-image }
+
+We'll cover everything from configuration to testing.
+
+## Step 1: Configure Your Application
+
+To deploy your application, you need to prepare a `requirements.txt` file
+that lists all the dependencies necessary for your Taipy application.
+
+Here are the steps to create this file:
+
+1. In your project directory, create a new file named `requirements.txt`.
+2. List all the dependencies, one per line, as shown in the example below
+3. Create a zip archive containing your project folder, including your application’s 
+`requirements.txt` and `main.py` files.
+
+```py
+taipy
+scikit-learn
+statsmodels
+pandas
+```
+
+!!! Warning
+    Using Windows-specific libraries is not supported, as the machines operate 
+    on a Linux environment.
+
+
+!!! Note
+    For uploading large data sets or sizable files, please use the `/home/user/data`
+     directory. This location preserves your data across deployments. To transfer your 
+     files, establish a connection to your machine using 
+     [SSH or Filezilla](#connect-with-ssh-or-filezilla) and upload your data 
+     directly to this directory.
```

Comment:
`/home/user/data` will not be supported in the feature due to the new data management on taipy-cloud. So please do not document it.